mlos_bench/mlos_bench/environments/status.py

```diff
@@ -46,16 +50,17 @@ def is_good(self) -> bool:
             Status.SUCCEEDED,
         }
 
+    # Class based accessor method to avoid circular import
+    @staticmethod
+    def completed_statuses() -> set["Status"]:
```

Comment:
Preparing this set to be reuseable in some SQL statements.

Reply:
maybe use `frozenset` for const sets?